### ISSUE TITLE
Traceback on exit because the Bugzilla plugin tries to use pynotify even if not available

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#111]
In Fedora 22, I start GTG:

```
$ gtg
2015-05-26 23:55:56,095 - WARNING - __init__:__init__:74 - Backend backend_evolution could not be loaded: No module named evolution
2015-05-26 23:55:56,102 - WARNING - __init__:__init__:74 - Backend backend_launchpad could not be loaded: No module named launchpadlib.launchpad
2015-05-26 23:55:56,104 - WARNING - __init__:__init__:74 - Backend backend_mantis could not be loaded: No module named suds.client
/usr/bin/notify-send
```

...but when I close the program normally, I get the traceback catcher dialog that tells me gtg choked on something. These are the details you can find in the terminal:

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib/python2.7/site-packages/GTG/plugins/bugzilla/notification.py", line 64, in uinit_pynotify
    pynotify.uninit()
NameError: global name 'pynotify' is not defined

Error in sys.exitfunc:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib/python2.7/site-packages/GTG/plugins/bugzilla/notification.py", line 64, in uinit_pynotify
    pynotify.uninit()
NameError: global name 'pynotify' is not defined
```

This seems quite unimportant and the notification feature should just be disabled when unavailable, or turned into a proper dependency.

**Comment by tomspur** [comment]
This should be fixed in commit 16ec3ade. Alternatively, you could install `notify-python` to fix this.
See [this](https://bugzilla.redhat.com/show_bug.cgi?id=1074757) Fedora bug report.

**Comment by izidormatusov** [comment]
As @tomspur said, it is fixed in master. We plan to release next version of GTG after @jakubbrindza  finishes his GSoC.
